data/config-schema/src/main/resources/net/corda/schema/configuration/flow/1.0/corda.flow.json

```diff
@@ -109,6 +109,13 @@
           "minimum": 10000,
           "maximum": 2147483647,
           "default": 600000
+        },
+        "maxPayloadSize": {
+          "description": "The maximum size of a payload in bytes that is allowed to be sent to a counterparty at once.",
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 2147483647,
+          "default": 104857600
```

Comment:
It's probably a good idea to add this to the kdoc comments on the `send` APIs, if you update those, letting people know that the max size is defined by `SESSION_FLOW_MAX_PAYLOAD` and it defaults to 100MB